test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/model/data/json/JSONUtilsTest.java

```diff
@@ -81,7 +81,7 @@ public void parseDateTest() {
 	public void getStringTest() {
 		Assert.assertEquals("London", JSONUtils.getString(map, "Location"));
 		Assert.assertEquals(null, JSONUtils.getString(map, "Time"));
-	}
+    }
```

Comment:
It's better to leave it untouched or format the whole file. Now it brings inconsistency.